AppCenterIdentity/AppCenterIdentity/MSIdentity.m

```diff
@@ -364,12 +366,14 @@ - (NSString *)retrieveAuthToken {
   return authToken;
 }
 
-- (void)removeAuthToken {
+- (BOOL)removeAuthToken {
   NSString *authToken = [MSKeychainUtil deleteStringForKey:kMSIdentityAuthTokenKey];
   if (authToken) {
     MSLogDebug([MSIdentity logTag], @"Removed auth token from keychain.");
+    return YES;
```

Comment:
Just `return authToken != nil` at the end of the method.

Reply:
It seemed to me it's more clear. I'll change it, not a big deal